R/calculate-pk-parameters-task.R

```diff
@@ -0,0 +1,37 @@
+#' @title CalculatePKParametersTask
+#' @docType class
+#' @description  Calculating PK parameters task settings for Reporting Engine
+#' @field resultsFileName string with no file extension.  Name of CSV file to which simulation results will be saved.
+#' @section Methods:

```

Comment:
@pchelle @abdullahhamadeh 
For new classes, please update to new R6 doc conventions
e.g. you can document methods right away where the method is defined

See ospsuite-r for example
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/develop/R/population.R